src/reducers/authReducer.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+interface AuthState {
+  user: {
+    token: string | null;
+  };
+}
+
+const initialState: AuthState = {
+  user: {
+    token: null,
+  },
+};
+
+const authSlice = createSlice({
+  name: 'counter',
```

Comment:
@SergeInGithub @musllim @shyakadavis. Remember To use appropriate naming conventions
`counter` ???.

Reply:
@musllim